esmvalcore/preprocessor/_regrid.py

```diff
@@ -260,42 +367,61 @@ def extract_point(cube, latitude, longitude, scheme):
 
 
 def regrid(cube, target_grid, scheme, lat_offset=True, lon_offset=True):
-    """
-    Perform horizontal regridding.
+    """Perform horizontal regridding.
+
+    Note that the target grid can be a cube (:py:class:`~iris.cube.Cube`),
+    path to a cube (``str``), a grid spec (``str``) in the form
+    of `MxN`, or a ``dict`` specifying the target grid.
+
+    For the latter, the ``target_grid`` should be a ``dict`` with the
+    following keys:
+
+    - ``lon_start`` is the longitude of the first grid cell.
```

Comment:
I'd go with "longitude value at the center of the first grid cell" so you can explain what the step is as the "distance between grid cell centers" - I find it more intuitive this way

Reply:
Will do 👍